Library/ShippingMethod.cs

```diff
@@ -0,0 +1,154 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Xml;
+
+namespace Recurly
+{
+    public class ShippingMethod : RecurlyEntity
+    {
+        public string Code { get; set; }
+        public string Name { get; set; }
+        public string AccountingCode { get; set; }
+        public string TaxCode { get; set; }
+        public DateTime CreatedAt { get; private set; }
+        public DateTime UpdatedAt { get; private set; }
+
+        internal const string UrlPrefix = "/shipping_methods/";
+
+        private string memberUrl()
+        {
+            return UrlPrefix + Code;
+        }
+
+        #region Constructors
+
+        internal ShippingMethod()
+        {
+        }
+
+        internal ShippingMethod(XmlTextReader xmlReader)
+        {
+            ReadXml(xmlReader);
+        }
+
+        #endregion
+
+        #region Read and Write XML documents
+
+        internal override void ReadXml(XmlTextReader reader)
+        {
+            while (reader.Read())
+            {
+                // End of shipping_method element, get out of here
+                if (reader.Name == "shipping_method" && reader.NodeType == XmlNodeType.EndElement)
+                    break;
+
+                if (reader.NodeType != XmlNodeType.Element) continue;
+
+                switch (reader.Name)
+                {
+                    case "code":
+                        Code = reader.ReadElementContentAsString();
+                        break;
+
+                    case "name":
+                        Name = reader.ReadElementContentAsString();
+                        break;
+
+                    case "accounting_code":
+                        AccountingCode = reader.ReadElementContentAsString();
+                        break;
+
+                    case "tax_code":
+                        TaxCode = reader.ReadElementContentAsString();
+                        break;
+
+                    case "created_at":
+                        CreatedAt = reader.ReadElementContentAsDateTime();
+                        break;
+
+                    case "updated_at":
+                        UpdatedAt = reader.ReadElementContentAsDateTime();
+                        break;
+                }
+            }
+        }
+
+        internal override void WriteXml(XmlTextWriter xmlWriter)
+        {
+            xmlWriter.WriteStartElement("shipping_method"); // Start: shipping_method
```

Comment:
I could have just thrown a `NotImplementedException` here because there is no `POST /shiping_method` endpoint at this time. But I have a feeling that one day it may be added, and this will make it that much quicker to implement support for that endpoint